### DISCUSSION TITLE
Advances administration protocol does not work correctly

### DISCUSSION THREAD

**Discussion by wschm** [Open-Systems-Pharmacology/Forum#1720] [Questions & Problems]
I tried to set up an advanced protocol based on a user defined application (drug appearing in lung cells). The protocol shloud consist of three dosing periods of 28 days with itermittant drug holidays of 8 weeks. For the first dosing interval everything seems to be OK while for the additionally introduced later intervals either the application rout is different or teh formulation (in this case first order) is not considered. At least the resulting curve shape is different from that in the first 28 applications.
So far I only found the workaround of defining daily application and setting all days in the drug holidays to dose 0, what is very nasty.

**Comment by Yuri05** [comment]
Can you upload 
a) a screenshot of your initially defined advanced protocol (expand ALL nodes)?
b) a screenshot of your formulation mapping in "Create Simulation" configuration?
